README.md

```diff
@@ -78,6 +78,18 @@ Flink package layout:
     └── flink          // flink client jars
 ```
 
+MR package layout:
+```
+    ├── RELEASE                         
+    ├── bin                             
+    ├── conf                            
+    ├── jars           // common jars for master and worker                        
+    ├── master-jars                     
+    ├── worker-jars                     
+    ├── sbin
+    └── mr          // mr client jars
```

Comment:
I'm wondering if we can merge the layout, and simply comment the engine's client folder as optional.